inc/passwords/namespace.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace HM\Platform\Security\Passwords;
+
+use function HM\Platform\get_config;
+use WP_Error;
+use ZxcvbnPhp\Zxcvbn;
+
+/**
+ * Bootstrap.
+ */
+function bootstrap() {
+	add_action( 'check_passwords', __NAMESPACE__ . '\\enforce_password_strength', 10, 3 );
+	add_action( 'admin_print_styles-user-edit.php', __NAMESPACE__ . '\\hide_weak_password_prompt' );
+	add_action( 'admin_print_styles-profile.php', __NAMESPACE__ . '\\hide_weak_password_prompt' );
+}
+
+/**
+ * Get minimum required strength for passwords.
+ *
+ * @return int
+ */
+function get_minimum_strength() {
+	$config = get_config()['modules']['security'];
+
+	/**
+	 * Filter the minimum required password strength.
+	 *
+	 * Filter the required strength for a password. These scores correspond to
+	 * the following UI strings:
+	 *
+	 * - 1 = "Very Weak"
+	 * - 2 = "Weak"
+	 * - 3 = "Medium"
+	 * - 4 = "Strong"
+	 *
+	 * @param int $minimum_strength Minimum required strength. Defaults is 3.
+	 */
+	return apply_filters( 'hm-platform.security.passwords.minimum_strength', $config['minimum-password-strength'] );
+}
+
+/**
+ * Enforce minimum password strength during password check.
+ *
+ * @param string $user_login User login slug
```

Comment:
This documentation is not as elaborate as the one in Core, so when in doubt copy it over. Especially the part about passing via reference is missing.

Reply:
I don't think there's a need to specific that the variables are by-reference, as it's in the code itself. Seems redundant?